src/main/scss/templates/_friends.scss

```diff
@@ -0,0 +1,53 @@
+// Friends styling
+.friends {
+  display: block;
+  padding: $size-base * 2;
+
+  .card {
+    background-color: $color-white;
+    padding: $size-base * 2;
+    border-radius: $size-base;
+    margin-right: $size-base * 3;
```

Comment:
Because we're dealing with the same size and shape here, let's try to make this margin symmetrical. `margin: $size-base * 1.5` or round to a whole number.